src/Covid19Dashboard/index.jsx

```diff
@@ -285,6 +285,7 @@ class Covid19Dashboard extends React.Component {
                         key={i}
                         chartsConfig={carouselConfig}
                         {...this.props}
+                        popupOnClick
```

Comment:
did you mean for this to be parametrizable through the config or something?

Reply:
it is hard-coding it to be true in in instance, it is not configurable 

would you prefer i write it out? to make it more obvious what it does

Reply:
it makes sense now that i read your answer to Zakir's comment!